sonumirror.md

```diff
@@ -0,0 +1,80 @@
+we saw that gamers wanted to stay updated on changes to game. Thus, we decided to leverage this platform to add two modules -- a 1-second-a-day module and a game patch notes module. Respectively, these would make it easier for people to build their 
+own 1-second-a-day videos and stay current on game updates.
+
+## Plan
+---Technologies
+
+In this project it can be broken down into three major parts:
+1. Smart Mirror construction
+2. 1-second-a-day module development
+3. Patch notes module development
+
+* *Smart Mirror Construction* * <br>
+smart mirror consists of five main components:
+1. A Monitor
+2. A Computer
+3. A Webcam
+4. A Two-way mirror
+5. A Frame
+<br>
+* *Frame Building Process* 
+
+Before building mirror, I had to give careful attention in researching and choosing the parts. Many
+options for components can conflict with other components due to physical or software constraints
+
+* *Monitor* * <br>
+I chose the LED monitor for multiple reasons. First, the monitor was 24" wide diagonally.
+This was large enough for a person to see most of their upper body in their reflection but small enough so
+our two-way mirror didn't have to be sized at an inordinately expensive size. Second, the bezel of the
+monitor was removable via a flathead screwdriver. This allows us to arrange the monitor's display flush
+against the two-way mirror, resulting in a sharper image
+<br>
+
+* *Rasberry Pie* * <br>
+The Raspberry Pi is a compact computer popular in many home internet-of-things projects. I chose
+the Raspberry Pi because of its small size and low cost.
+<br>
+
+* *Two-way Mirror* * <br>
+A smart mirror's mirror can be made of reflective film, glass, or acrylic. I decided to go with a ready-cut
+two-way mirrored acrylic from TapPlastics. Acrylic is superior to film, in that it allows more light from the
+monitor to shine through. At the same time, it is cheaper and less fragile than glass.
+<br>
+
+* *Frame: Glacier Bay 15 1/4in x 26in Medicine Cabinet* * <br>
+I choose the medicine cabinet as model frame. First, its mirror opening is the perfect size -- only slightly
+larger than my LED monitor. Second, its hollow body provides just enough room to store the monitor.
+Third, its built-in hinge allows us to easily open up the smart mirror and access/adjust its internal
+components
+<br>
+
+Just want to clarify that these websites I'll be using to buy the material.
+
+## Budget
+
+| Product           | Supplier/Link                         | Cost    |
+| ---------------   | ------------------------------------- | ------  |
+| Rasberry Pi       | https://amzn.eu/d/h6dXBEs             | $176.44 |
+| Webcam            | https://amzn.eu/d/548ZtSv             | $23.20  |
+| Frame             | https://amzn.eu/d/548ZtSv             | $10.00  |
+| Connectors        |                                       | $5.00   |
```

Comment:
please clarify where you're buying the connectors from! if the shop doesn't have a website, please just note it's name

Reply:
I will purchase the connectors from local store, from Gujarati Galli Hyderabad, here we have huge stores to purchase all the electornics needed. Thank you.

Reply:
> I will purchase the connectors from local store, from Gujarati Galli Hyderabad, here we have huge stores to purchase all the electornics needed. Thank you.

Hello sir did check the changes